### PR TITLE
feat: add slider limits to web

### PR DESCRIPTION
**Pull request by JacobDel**
Continues on abandoned PR: https://github.com/callstack/react-native-slider/pull/520

Summary:
---------

Add upperLimit support and lowerLimit support to web.

Test Plan:
----------

Also added an example to the web-example project.
https://github.com/callstack/react-native-slider/assets/38087818/32c09f40-cbc4-4dec-ba51-0cda036ef442

### CODE REVIEW

File: package/src/Slider.tsx [hunk 1]
```diff
@@ -240,12 +240,12 @@ const SliderComponent = (
   const lowerLimit =
     !!localProps.lowerLimit || localProps.lowerLimit === 0
       ? localProps.lowerLimit
-      : LIMIT_MIN_VALUE;
+      : Platform.select({ web: localProps.minimumValue, default: LIMIT_MIN_VALUE });
```
Comment (BartoszKlonowski) [inline package/src/Slider.tsx:243]: Currently, for now this is alright, however, we should aim for unified approach for all platforms.
So the options (for future) are:
1. Remove `LIMIT_MIN_VALUE` approach in favor of the minimum value
2. Use the number range threshold for every.

This is just a thought for future, doesn't concern this PR.

**Review by BartoszKlonowski — APPROVED** [review]
This looks great: simple and working. Thank you!
I only have one minor nit regarding naming, as I found it confusing at first and would like to make it a bit more readable - once you accept or anyhow adjust, it's totally ready to be merged.
Thank you 👍

**Review by BartoszKlonowski — COMMENTED** [review]
Some linting corrections.

**Comment by JacobDel** [comment]
I'm not sure how to fix the Lint issue

File: package/src/Slider.tsx [hunk 2]
```diff
@@ -240,12 +240,18 @@ const SliderComponent = (
...
+        });
 
   const upperLimit =
     !!localProps.upperLimit || localProps.upperLimit === 0
       ? localProps.upperLimit
-      : LIMIT_MAX_VALUE;
+      : Platform.select({
+          web: localProps.maximumValue,
+          default: LIMIT_MAX_VALUE,
+      });
```
Comment (BartoszKlonowski) [inline package/src/Slider.tsx:254]:
```suggestion
        });
```

**Comment by BartoszKlonowski** [comment]
@JacobDel Normally you should run `npx eslint --fix 'file'` for the 'file' you would like to lint.
I decided to do that through GH PR interface, hoping it will be faster, but I made the simple mistake of missing indentation.
I already fixed it.

**Comment by JacobDel** [comment]
Do you have an estimate when this would be released?

**Comment by BartoszKlonowski** [comment]
Hello @JacobDel! Most probably next Friday (23.02).

**Comment by JacobDel** [comment]
Hi, any updates on this? Is there a way to build and use it locally?

**Pull request merged**